tgui/packages/tgui/interfaces/NtosConfiguration.js

```diff
@@ -26,8 +25,8 @@ export const NtosConfiguration = (props, context) => {
           <LabeledList>
             <LabeledList.Item
               label="Battery Status"
-              color={!battery_exists && 'average'}>
-              {battery_exists ? (
+              color={!battery && 'average'}>
+              {battery ? (
```

Comment:
You should remove the initializer from `battery = {}`, from a few lines above, because it kind of implies it can't be null, and it can mess with the null check here.